test/typescript/tsconfig.json

```diff
@@ -1,8 +1,11 @@
 {
-  "files": ["test.ts"],
   "compilerOptions": {
-    "esModuleInterop": true, // required for import cds from '@sap/cds'
-    "types": [ "node" ], // to ignore @types/mocha and others
-    "outDir": "_out"
+    "module": "ES2022",
+    "target": "ES2022",
+    // "strict": true,
```

Comment:
Enabling strict mode would reveal some issues with the test code.
We should fix them though for better confidence.